docs/docs/guide/6_faq/index.md

```diff
@@ -11,4 +11,39 @@ The preferred solution is to use subdomains for each app. For example, you can u
 ## How can I use components that resemble the original components in the administration?
 While it is not possible to use the exact same components in the Shopware administration, there is a component library called Meteor Component Library that offers similar components. The Shopware administration components are not native Vue components because they have extension capabilities, Twig templates, and other features that cannot be directly used. However, by utilizing the Meteor Component Library, you can achieve a native look and feel for your app that seamlessly integrates with the original Shopware administration.
 
-To access the Meteor Component Library, visit the following link: https://github.com/shopware/meteor-component-library
\ No newline at end of file
+To access the Meteor Component Library, visit the following link: https://github.com/shopware/meteor-component-library
+
+## How can I use snippets for multi-language in my app?
+
+You can manage all text rendered within your app's [locations](../4_concepts/locations.md) with a translation plugin of your choice. If you're utilizing Vue.js as your frontend framework you can use the i18n plugin. Additionally, to ensure consistency between your app and the administration interface, you can synchronize language changes by [subscribing to them through the context API](../2_api-reference/context.md#subscribe-on-language-changes).
+
+For text elements outside your locations, such as titles within [component sections](../4_concepts/component-sections.md), you can employ snippet files within your app. Here's a how to accomplish this:
+
+1. **Create Snippet Files:** Begin by generating a snippet file for each supported language within your app. These files should reside in the `Resources/app/administration/snippet` directory. Naming conventions follow the language code format, for instance, `en-GB.json` for English language support. The file structure mirrors that of administration snippets.
```

Comment:
```suggestion
1. **Create Snippet Files:** Begin by generating a snippet file for each supported language within your app. These files should reside in the `Resources/app/administration/snippet` directory. Naming conventions follow the language code format, for instance, `en-GB.json` for English language support. The file structure mirrors that of administration snippets. However it is impossible to overwrite Shopware Administration snippets.
```

Reply:
This is not possible, see https://developer.shopware.com/docs/guides/plugins/plugins/administration/adding-snippets.html#creating-snippet-files So it have to be 

```suggestion
1. **Create Snippet Files:** Begin by generating a snippet file for each supported language within your app. These files should reside in the `Resources/app/administration/snippet` directory. Naming conventions follow the language code format, for instance, `en-GB.json` for English language support. The file structure mirrors that of administration snippets. However it is not impossible to overwrite Shopware Administration snippets.
```